devsetup/scripts/bgp-interfaces-setup.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# set -x
+
+if [ "$EUID" -eq 0 ]; then
+    echo "Please do not run as root."
+    exit
+fi
+
+sudo virsh destroy crc
+
+cat << EOF > /tmp/pci1.xml
+<controller type='pci' index='7' model='pcie-root-port'>
+  <model name='pcie-root-port'/>
+  <target chassis='7' port='0x16'/>
+  <alias name='pci.7'/>
+  <address type='pci' domain='0x0000' bus='0x00' slot='0x02' function='0x6'/>
+</controller>
+EOF
+cat << EOF > /tmp/pci2.xml
+<controller type='pci' index='8' model='pcie-root-port'>
+  <model name='pcie-root-port'/>
+  <target chassis='8' port='0x15'/>
+  <alias name='pci.8'/>
+  <address type='pci' domain='0x0000' bus='0x00' slot='0x02' function='0x7'/>
+</controller>
+EOF
+
+cat << EOF > /tmp/net1.xml
+<interface type='network'>
+  <source network='default'/>
+  <model type='virtio'/>
+  <address type='pci' domain='0x0000' bus='0x06' slot='0x00' function='0x0'/>
+</interface>
```

Comment:
should this be the same as in https://github.com/openstack-k8s-operators/install_yamls/blob/main/devsetup/Makefile#L107 ? would it make sense to make this script generic to run by default what we currently have in the Makefile and in the BGP case also add the other?

Reply:
I think the problem we had was that we needed to add extra pci slots. You are suggesting to add the pci slots in the BGP case too, right?

Reply:
yes, add the pci slots in case of bgp, but not in the default one